### PR TITLE
DOCS-389 solved code duplication: App Router vs. Pages Router example

### PR DESCRIPTION
**Pull request by alexisintech**
This example had the exact same code as the Pages Router example. This PR fixes the App Router code example.

### CODE REVIEW

**Review by JacobMGEvans — APPROVED** [review]
awwww yeah, App Router for life. 😆

File: docs/components/user/user-button.mdx [hunk 1]
```diff
@@ -40,18 +40,20 @@ In this example, `<UserButton />` is mounted inside a header component, which is
...
 
-      function MyApp({ pageProps }) {
+      export default function RootLayout({ children }: { children: React.ReactNode }) {
         return (
-          <ClerkProvider {...pageProps}>
-            <Header />
-          </ClerkProvider>
+          <html lang="en">
+            <ClerkProvider>
+              <Header />
```
Comment (BRKalow) [inline docs/components/user/user-button.mdx:50]: Layouts need to render their children, which will be the matching page component.

```suggestion
              <Header />
              {children}
```

File: docs/components/user/user-button.mdx [hunk 2]
```diff
@@ -40,18 +40,20 @@ In this example, `<UserButton />` is mounted inside a header component, which is
...
+            </ClerkProvider>
+          </html>
         );
       }
 
-      export default MyApp;
+      export default RootLayout;
```
Comment (BRKalow) [inline docs/components/user/user-button.mdx:56]: The layout is already exported above.

```suggestion
```

Comment (alexisintech) [reply docs/components/user/user-button.mdx:56]: I forgot to delete this line when I moved the export 😭

File: docs/components/user/user-button.mdx (hunk 1 above)
Comment (alexisintech) [reply docs/components/user/user-button.mdx:50]: great catch!!

**Pull request merged**